### ISSUE TITLE
Allow searching by date range (since: and until:)

### ISSUE THREAD

**Issue by oznogon** [bluesky-social/social-app#1523]
**Is your feature request related to a problem? Please describe.**

Bluesky doesn't allow searching or filtering posts by date range.

**Describe the solution you'd like**

Search operators that allow returning results from a date range.

To return all posts containing `football` since 2 Jan 2023:

```
football since:2023-01-02
```

Posts before 2 Jan 2023:

```
football until:2023-01-02
```

These operators can be combined to select a date range, and combined with any other search operators to further filter results.

```
football since:2023-01-02 until:20203-06-02
```

**Describe alternatives you've considered**

Can't find any.

**Additional context**

This is a request for feature parity with other similar social posting services; see also https://github.com/bluesky-social/social-app/issues/1143, https://github.com/bluesky-social/social-app/issues/849, #1522.

**Comment by oznogon** [comment]
This appears to be implemented. I can't pinpoint what implemented it, but I found bluesky-social/indigo/pull/643 and it's documented in a May 2024 blog post: https://bsky.social/about/blog/05-31-2024-search

I can't find any mention of these search operators in the public social app docs.

**Comment by oznogon** [comment]
#4093 proposes a way to document these in-app.
